OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -60,15 +60,19 @@ public AttackBase(Actor self, AttackBaseInfo info)
 			this.self = self;
 		}
 
-		void INotifyCreated.Created(Actor self)
+		protected override void Created(Actor self)
```

Comment:
I think this one should be the virtual function - this one has no parent class. (Which also means you don't need base.Created here).